example/steps/p4est_step3.c

```diff
@@ -1489,9 +1489,9 @@ main (int argc, char **argv)
 
   /* read command line options */
   opt = sc_options_new (argv[0]);
-  sc_options_add_bool (opt, 'C', "write checkpoints", &step3_checkpoint, 0,
+  sc_options_add_bool (opt, 'C', "write-checkpoints", &step3_checkpoint, 0,
```

Comment:
Singula please. :)

Reply:
Right, we only write once at the end of the simulation.